modules/material/locales/en/LC_MESSAGES/material.po

```diff
@@ -182,6 +182,9 @@ msgstr "your supervisor"
 msgid "{mfa:recovery_sent}"
 msgstr "A temporary code was sent to your recovery contact."
 
+msgid "{mfa:recovery_input}"
+msgstr "Enter code"
```

Comment:
Sorry I recommended removing this and you had to add it back.

Reply:
Not a problem. Predicting the future is still hard. 😉